tests/test_repo_interactions.py

```diff
@@ -80,6 +91,23 @@ def test_chartpress_run(git_repo, capfd, base_version):
         f"Updating testchart/values.yaml: image: testchart/testimage:{reset_tag}" in out
     )
 
+    # verify usage of --config
+    with temporary_copy("chartpress.yaml") as temp:
```

Comment:
It looks like this copy isn't used anywhere else, so can it be a separate testcase? Or parameterise the whole test (so the whole of test_chartpress_run is run with chartpress.yaml and a custom-named config file)?

Reply:
Great idea to parameterize